src/PhpSpreadsheet/Reader/Xlsx.php

```diff
@@ -2562,13 +2562,15 @@ private function readColumnsAndRowsAttributes(SimpleXMLElement $xmlSheet, Worksh
             }
         }
 
+        $readFilter = (\get_class($this->getReadFilter()) !== DefaultReadFilter::class ? $this->getReadFilter() : null);
+
         // set columns/rows attributes
         $columnsAttributesSet = [];
         $rowsAttributesSet = [];
         foreach ($columnsAttributes as $coordColumn => $columnAttributes) {
-            foreach ($rowsAttributes as $coordRow => $rowAttributes) {
-                if ($this->getReadFilter() !== null) {
-                    if (!$this->getReadFilter()->readCell($coordColumn, $coordRow, $docSheet->getTitle())) {
+            if ($readFilter !== null) {
+                foreach ($rowsAttributes as $coordRow => $rowAttributes) {
+                    if (!$readFilter->readCell($coordColumn, $coordRow, $docSheet->getTitle())) {
```

Comment:
Is there any measurable benefit to caching the title as well?

Reply:
Good question, my xdebug tracing did not show excessive `getTitle()` calls, but maybe I just overlooked them.